scss/style.scss

```
@@ -0,0 +1,399 @@
+@import url("https://fonts.googleapis.com/css?family=Aleo:regularc&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Josefin+Sans:300,regular,700 &display=swap");
+*,
+::before,
+::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.travel {
+  &__header {
+    background: url(../img/HeaderBackground.png) no-repeat;
+    background-size: cover;
+    width: 100%;
+  }
+  &__navigation {
+    background: #5373fd;
+  }
+  &__why {
+    background: url(../img/WhyBackground.png) no-repeat;
+    background-size: cover;
+    width: 100%;
+  }
+  &__steps {
+    background: #333333;
+  }
+  &__book1 {
+    background: url(../img/render-1477041_12801.png) center no-repeat;
+    background-size: cover;
+    width: 100%;
+  }
+  &__book2 {
+    background: url(../img/spain-1859434_12801.png) center no-repeat;
+    background-size: cover;
+    width: 100%;
+  }
+  &__whattouristsay {
+    background: url(../img/BG2.png) center no-repeat;
+    background-size: cover;
+    width: 100%;
+  }
+  &-footer {
+    background: #000000;
+  }
+}
+.travel-text {
+  &__verysmall {
+    padding-top: 44px;
+    padding-left: 32px;
+    padding-bottom: 16px;
+    font-family: Josefin Sans;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 20px;
+    line-height: 28px;
+    letter-spacing: 0.1em;
+    color: #ffffff;
+  }
+  &__small {
+    padding-left: 32px;
+    padding-bottom: 24px;
+    font-family: Josefin Sans;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 16px;
+    line-height: 28px;
+    letter-spacing: 0.1em;
+    color: #ffffff;
+  }
+  &__submiddle {
+    padding-top: 32px;
+    margin-bottom: 32px;
+    font-family: Josefin Sans;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.2em;
+    color: #ffffff;
+  }
+  &__middle {
+    padding-left: 32px;
+    padding-bottom: 16px;
+    font-family: Josefin Sans;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+    text-transform: uppercase;
+    color: #ffffff;
+  }
+  &__bigger {
+    font-family: Aleo;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 36px;
+    line-height: 40px;
+    text-align: center;
+    text-transform: capitalize;
+    color: #ffffff;
+  }
+  &__bigger-last {
+    font-family: Aleo;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 36px;
+    line-height: 40px;
+    text-align: center;
+    text-transform: capitalize;
+    color: #ffffff;
+    padding-top: 120px;
+    margin-bottom: 60px;
+  }
```

тут хорошо, что ты повыделял разные стили для текста в разные классы, но лучше паддинги и маржины сдесь не вписывать, а ставить только относящееся к тексту
```suggestion
  &__bigger-last {
    font-family: Aleo, serif;
    font-size: 36px;
    line-height: 40px;
    text-transform: capitalize;
    color: #ffffff;
  }
```